### PR TITLE
Package ipysheet

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: recipes/ipysheet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  license_file: LICENSE
+  summary: 'Spreadsheet widget for Jupyter'
+  description: 'Spreadsheet widget for Jupyter'
+  dev_url: https://github.com/QuantStack/ipysheet
+  dev_url: https://github.com/QuantStack/ipysheet
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - MaartenBreddels
```
Comment (beckermr) [inline recipes/ipysheet/meta.yaml]: @maartenbreddels Should comment here that they are ok maintaining this package.

**Comment by maartenbreddels** [comment]
All good :+1:

(from mobile phone)

**Comment by SylvainCorlay** [comment]
@beckermr this is ready to go!

**Comment by beckermr** [comment]
Weird the travis status did not sync for me.

**Pull request merged**